src/auxiliary/mod.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This module is about interaction with auxiliary block chain.
+
+use futures::Future;
+use rlp;
+use sha3::{Digest, Keccak256};
+use web3::contract::Options;
+use web3::types::{Address, H160, H256};
+
+use super::ethereum::Ethereum;
+use ethereum::types::{Block, Error, ErrorKind};
```

Comment:
Why is one with `super` and one without? 🤔

Reply:
I will make repo consistent to use `super` or explicit module name